pcdsdevices/mirror.py

```diff
@@ -815,37 +871,25 @@ def format_status_info(self, status_info):
 """
 
 
-class TwinCATMirrorStripe(TwinCATStatePMPS):
+class FFMirrorZ(FFMirror):
     """
-    Subclass of TwinCATStatePMPS for the mirror coatings.
+    Fixed Focus Kirkpatrick-Baez Mirror with Z axis.
 
-    Unless most TwinCATStatePMPS, we have:
-    - Only in_states
-    - No in_states block the beam
-
-    We also clear the states_list and set _in_if_not_out to True
-    to automatically pick up the coatings from each mirror enum.
-    """
-    states_list = []
-    in_states = []
-    out_states = []
-    _in_if_not_out = True
-    config = UpCpt(state_count=2)
-
-    @property
-    def transmission(self):
-        """The mirror coating never blocks the beam."""
-        return 1
+    1st gen Toyama designs with LCLS-II Beckhoff motion architecture.
 
+    Parameters
+    ----------
+    prefix : str
+        Base PV for the mirror.
 
-class CoatingState(Device):
+    name : str
+        Alias for the device.
     """
-    Extra parent class to put "coating" as the first device in order.
+    # Motor components: can read/write positions
+    z = Cpt(BeckhoffAxisNoOffset, ':MMS:Z', kind='hinted')
```

Comment:
There's probably some way to re-order the components for the typhos screen. I'm going to dig around in ophyd for a bit.

Reply:
It looks like there is absolutely a sneaky way to rearrange the screen. See #994 for a proof of concept that should probably be finalized and put in as a utility.